cli/src/main/scala/pravda/cli/Config.scala

```diff
@@ -22,6 +23,9 @@ object Config {
 
   final case class GenAddress(output: Option[String] = None) extends Config
 
+  final case class GenDocs(outDir: String = "docs", mainPageName: String = "main.md", cl: CommandLine[Config])
```

Comment:
Wow! You're made Markdown generation as part of CLI? I'm sure that the functionality shouldn't be visible for our users. It's better to make dedicated runner (alternative main-class of the `cli` module) which will print Markdown.